src/opm/io/eclipse/rst/state.cpp

```diff
@@ -174,6 +176,55 @@ void RstState::add_msw(const std::vector<std::string>& zwel,
     }
 }
 
+void RstState::add_udqs(const std::vector<int>& iudq,
+                        const std::vector<std::string>& zudn,
+                        const std::vector<std::string>& zudl,
+                        const std::vector<double>& dudw,
+                        const std::vector<double>& dudg,
+                        const std::vector<double>& dudf) {
+
+    for (auto udq_index = 0; udq_index < this->header.num_udq(); udq_index++) {
+        const auto& name = zudn[udq_index * UDQDims::entriesPerZUDN()];
+        const auto& unit = zudn[udq_index * UDQDims::entriesPerZUDN() + 1];
+        auto define_int =  iudq[udq_index * UDQDims::entriesPerIUDQ()];
+
+        if (define_int == 2) {
+            auto zudl_begin = zudl.begin();
+            auto zudl_end = zudl.begin();
+            std::advance( zudl_begin, udq_index * UDQDims::entriesPerZUDL() );
+            std::advance( zudl_end, (udq_index + 1) * UDQDims::entriesPerZUDL() );
+            auto udq_define = std::accumulate(zudl_begin, zudl_end, std::string{}, std::plus<std::string>());
+            if (udq_define[0] == '~')
+                udq_define[0] = '-';
+
+            this->udqs.emplace_back(name, unit, udq_define);
+        } else
+            this->udqs.emplace_back(name, unit);
+
+        auto& udq = this->udqs.back();
+        if (udq.var_type == UDQVarType::WELL_VAR) {
+            for (std::size_t well_index = 0; well_index < this->wells.size(); well_index++) {
```

Comment:
You need `<cstddef>` (or one of a few [other headers](https://en.cppreference.com/w/cpp/types/size_t)) to bring a declaration of `std::size_t` into scope.